torch_xla/core/xrt_run_server.py

```diff
@@ -6,6 +6,7 @@
 import sys
 
 from pathlib import Path
+os.environ['INIT_LAZY_BACKEND'] = '0'
```

Comment:
Similar to the earlier comment -- also because I don't know why this wasn't done before and needed now. Maybe something like, `// disable INIT_LAZY_BACKEND in xrt_run_server because ...` could help us, otherwise, this could become a tribal knowledge.

Reply:
well the right fix is to make sure `_init_lazy_backend`'s corresponding key registration function to only trigger once per process. This fix is not enough, but a quick workaround since it is less intrusive.

Reply:
> this is a 1.13 only quick hack, I will fix it properly in the master.

ditto